cachito/workers/pkg_managers/npm.py

```diff
@@ -68,6 +73,11 @@ def _get_deps(package_lock_deps, file_deps_allowlist, _name_to_deps=None):
         # output of this function to download the dependencies will ignore this dependency.
         if info["version"].startswith("file:") and name in file_deps_allowlist:
             log.info("The dependency %r is an allowed exception", info)
+        #
+        elif info["version"].startswith("file:") and package_lock_lockfileversion >= 2:
```

Comment:
If the workspaces is not empty, do we need to care about the lock file version?
Can we just check it before calling `_get_deps` whether the lockfile consistent or not and then pass only `workspaces`?

Reply:
good point, thanks

Reply:
Or maybe you could make workspaces an optional parameter:
```
def _get_deps(package_lock_deps, file_deps_allowlist, workspaces=None, _name_to_deps=None):
```
And only fill it when lockfile_version >=2. Workspaces are a feature that should only be expected on v2, so I see no trouble reflecting this in the code. 

Or we could also go with Serhii's suggestion and ignore the lockfile_version altogether, just be sure there's no corner case we're ignoring here. I think the baseline is avoiding one additional parameter being passed down to this function.

Reply:
Oops, I forgot to push the changes. I changed the parameters exactly like you have it and added the check for the version in `get_package_and_deps`.